languages/elixir/exercises/concept/bitstrings/.meta/design.md

```diff
@@ -0,0 +1,29 @@
+## Learning objectives
+
+- Know what a bitstring is
+- Know how to specify a bitstring literal
+- Know about the `<<>>` special form
+- Know about the `::` binary size modifier
+- Know how to construct a bitstring
+- know how to pattern match a bitstring
+- learn about accumulators and TCO
+
+## Out of scope
+
+- binaries
+- strings
+
+## Concepts
+
+- `bitstrings`
+- `tail-recursion`
+
+## Prerequisites
+
+- `recursion`
+- `charlists`
+- `pattern-matching`
+
+## Analyzer
+
+-
```

Comment:
We should probably check that `Enum` is not used.

Reply:
I'm reasonably sure Enum could not be used effectively outside of traversing the charlist in the encoding function.  But would still require students to construct the bitstring as shown in the introduction -- so i don't think it would defeat the purpose of the exercise if students were to use it. Have we run into a situation like this -- where using an out-of-scope function wouldn't affect the learning outcome? What did we do then?

Checking for list comprehensions might also be useful since they can work with both lists and bitstrings.

Reply:
I'll add it for now